cpp/bench/sg/rf_regressor.cu

```diff
@@ -100,7 +100,7 @@ std::vector<RegParams> getInputs() {
                                      //   anyway changed below
                   32,                // n_bins
                   1,                 // split_algo
-                  3,                 // min_rows_per_node
+                  3,                 // min_samples_leaf
```

Comment:
why does this not have the `min_samples_split` param here, similar to the above?

Reply:
Fixed